lib/async_hooks.js

```diff
@@ -480,4 +496,6 @@ module.exports = {
   emitBefore: emitBeforeS,
   emitAfter: emitAfterS,
   emitDestroy: emitDestroyS,
+  // internal
+  _getTypeName,
```

Comment:
If this is meant for internal only, I would prefer that it be named with a Symbol rather than introducing a new `_`-prefixed property. Or, better yet, move it to an appropriate `lib/internal` location.